pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -14,17 +14,18 @@
 , libosmium
 , protozero
 , rapidjson
+, testers
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "osm2pgsql";
-  version = "1.7.0";
+  version = "1.7.1";
 
   src = fetchFromGitHub {
     owner = "openstreetmap";
-    repo = pname;
-    rev = version;
-    hash = "sha256-MWJzCZdqvy/nH1Doj0fmGuzTubaHDnPOED7qgzvJ3ZU=";
+    repo = "osm2pgsql";
```

Comment:
I guess hardcoding the repo like you do here is preferred over reusing `pname`, so that overriding `pname` won't interfere with what repo we fetch from?

Reply:
Indeed!